### PR TITLE
Lightbox email leadin

### PR DESCRIPTION
**Pull request by ribner**

### CODE REVIEW

File: packages/admithub-lightbox/package.js [hunk 1]
```diff
@@ -8,12 +8,17 @@ Package.onUse(function(api) {
   api.use([
     'accounts-base',
     'stylus',
-    'telescope:core@0.24.0',
+    'telescope:core',
```
Comment (AndrewMagliozzi) [inline packages/admithub-lightbox/package.js]: let's make sure that we don't accidentally upgrade to the next telescope version and break something down the line.  Though this change isn't a blocker.

File: packages/admithub-lightbox/lib/client/lightBox.js [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    } else {
+      addMask();
+      return true
+    }
+  }
+
+Template.layout.events({
+  'click a': function (e) {
+    Session.get('lightBoxPageViewCount') === undefined ? Session.set('lightBoxPageViewCount', 1 ) : Session.set('lightBoxPageViewCount', (Session.get('lightBoxPageViewCount') + 1 )) ;
+    if (Session.get('lightBoxPageViewCount') === 4 ) { Session.set('lightBoxPageViewSetting', checkShowSettings() ) }
```
Comment (AndrewMagliozzi) [inline packages/admithub-lightbox/lib/client/lightBox.js:41]: That's cool.  I never thought of setting the return value of a function as a session variable.

**Comment by AndrewMagliozzi** [comment]
If a user dismisses the newsletter, we should alert them that they can find the form later on their edit profile page.

File: packages/admithub-lightbox/package.js (hunk 1 above)
Comment (yourcelf) [inline packages/admithub-lightbox/package.js]: FYI: telescope dropped a new version today which has breaking changes for us. Please don't drop version pinning.

**Pull request closed without merging**